examples/virus_on_network/app.py

```diff
@@ -0,0 +1,134 @@
+import math
+
+import solara
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+from mesa.experimental import JupyterViz, make_text
+from virus_on_network.model import State, VirusOnNetwork, number_infected
+
+
+def agent_portrayal(graph):
+    def get_agent(node):
+        return graph.nodes[node]["agent"][0]
+
+    width = []  # edge width
```

Comment:
Maybe just call it edge_width in line with edge_color, so it doesn't need the comment?
```suggestion
    edge_width = []
```

Reply:
Fixed.